CONTRIBUTING.md

```diff
@@ -1 +1,63 @@
 The development guide is located under the [docs](docs/README.md) folder.
+
+# Contribution guidelines 
+
+Welcome to [oceanbase]! We're thrilled that you'd like to contribute. Your help is essential for making it better.
+
+## Getting Started
+
+Before you start contributing, please make sure you have read and understood our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+### Fork the Repository
+
+First, fork the [oceanbase repository](https://github.com/oceanbase/oceanbase) to your own GitHub account. This will create a copy of the project under your account.
+
+
+### Clone Your Own Repository
+```bash
+git clone https://github.com/`your-github-name`/oceanbase
+```
+### Navigate to the project directory 📁
+```bash
+cd oceanbase
+```
+Create a new branch for your feature or bug fix:
+```bash
+ git checkout -b feature-branch
+```
+
+> feature-branch is the name of the branch where you will be making your changes. You can name this whatever you want.
+
+Make your changes and commit them:
+```bash
+git add .
+git commit -m "Description of your changes"
+```
+Push your changes to your fork:
+```bash
+git push origin feature-branch
+```
+Finally Click on Create Pull request to contribute on this repository.
```

Comment:
fixed.
I use "Compare & pull request" instead of the source words.
<img width="950" alt="image" src="https://github.com/oceanbase/oceanbase/assets/5187215/b5662090-6ce6-472e-a5c4-24eee38196c7">